src/main/java/mainclasses/Ui.java

```diff
@@ -0,0 +1,41 @@
+package mainclasses;
+
+import java.util.Scanner;
+
+import exception.DukeException;
+
+public class Ui {
+    private Scanner sc;
+    private Parser parser;
+
+    /**
+     * UI class
+     * @param parser
+     */
+    public Ui(Parser parser) {
+        this.parser = parser;
+    }
+
+    /**
+     * Allows user to enter string text into the command line
+     */
+    public String executeInput(String userInput) {
+        String output = "";
+        try {
+            parser.checkInput(userInput);
+            output = parser.determineAction(userInput);
+        } catch (DukeException ex) {
+            System.out.println(ex.getMessage());
+        }
+        return output;
+    }
+
+    /**
+     * Initialise the first dialog from Duke
```

Comment:
Should it be "Intialises" rather than "Initialise"? A full stop "." is missing at the end of sentence.